include/jemalloc/internal/counter.h

```diff
@@ -0,0 +1,83 @@
+#ifndef JEMALLOC_INTERNAL_COUNTER_H
+#define JEMALLOC_INTERNAL_COUNTER_H
+
+#include "jemalloc/internal/mutex.h"
+
+typedef struct counter_accum_s {
+#ifndef JEMALLOC_ATOMIC_U64
+	malloc_mutex_t	mtx;
+	uint64_t	accumbytes;
```

Comment:
Since this disrupts the history anyways, might as well get rid of the mid-line tabs?

(Or maybe not; now that I have more jemalloc time, I might as well update https://github.com/jemalloc/jemalloc/pull/1250 assuming everyone is still on board).

Reply:
I'll remove the tabs anyway. For the record I'm fully on board and very much want clang format.